netlify.toml

```diff
@@ -1,32 +1,32 @@
 [build]
-  command = "npm run build"
-  publish = "dist"
+	command = "npm run build"
+	publish = "dist"
 
 
 # Redirects
 [[redirects]]
-  from = "/assets/PDFS/*"
-  to = "/PDFS/:splat"
-  status = 301
-  force = true
-  
+	from = "/assets/PDFS/*"
+	to = "/PDFS/:splat"
+	status = 301
+	force = true
+
 [[redirects]]
-  from = "/*"
-  to = "/404.html"
-  status = 404
+	from = "/*"
+	to = "/404.html"
+	status = 404
 
 # Build Post Processing
 [build.processing]
-  skip_processing = false
+	skip_processing = false
 [build.processing.css]
-  bundle = false
-  minify = true
+	bundle = false
+	minify = true
 [build.processing.js]
-  bundle = false
-  minify = true
+	bundle = false
+	minify = true
 [build.processing.html]
-  pretty_urls = true
+	pretty_urls = false
```

Comment:
Astro already does pretty URLs, I find having this setting on ends up with some strange results (going to `/about` redirects to `/about/`)